### ISSUE TITLE
model trained on VOC 2007 

### ISSUE THREAD

**Issue by akiradesu** [weichengkuo/DeepBox#1]
In your paper, you say that you evaluated DeepBox on both PASCAL VOC 2007 and COCO 2014 datasets. Would you please also make the model you trained on VOC 2007 trainval set publicly available?

**Comment by weichengkuo** [comment]
Hi,

Fast DeepBox is evaluated on COCO only -- no PASCAL model is available. But
for non-fast DeepBox there is PASCAL model. Is that what you want?

Sincerely,

Weicheng Kuo

Graduate Student Researcher,
University of California, Berkeley

**Comment by akiradesu** [comment]
No, but thanks all the same. Another question, I evaluated proposals generated using EdgeBox on PASCAL VOC 2007 test set and achieved a maximum recall of 76% with IoU = 0.7, which is far lower than 87% reported in Zitnick's paper. I used default values for all parameters. Any suggestions?

**Comment by weichengkuo** [comment]
Are you just using Edge boxes or DeepBox?

Sincerely,

Weicheng Kuo

Graduate Student Researcher,
University of California, Berkeley

**Comment by akiradesu** [comment]
Edge boxes. I'm using the code in [this repo](https://github.com/pdollar/edges).

**Comment by weichengkuo** [comment]
Then you should probably ask Piotr Dollar. However, I don't remember having
this problem when I run it on PASCAL VOC.

Sincerely,

Weicheng Kuo

Graduate Student Researcher,
University of California, Berkeley
